Plugin/CppApi/source/CoordinateConversion/CoordinateConversionController.cpp

```diff
@@ -381,11 +381,14 @@ bool CoordinateConversionController::isCaptureMode() const
 }
 
 /*!
-  \brief Sets the tool's capture mode to \a captureMode.
-
-  If \c true, the tool will convert a point set via a mouse click.
-  If \c false, the too will use the app's current location as the target point.
+  \qmlproperty bool CoordinateConversionController::captureMode
+  \brief The tool's capture mode
+  \list
+    \li \c true = convert a point set via a mouse click.
+    \li \c false = use the app's current location as the target point.
+  \endlist
  */
+
```

Comment:
@tdunn For the C++ toolkit components, I believe we need the public setters documented as well as the public properties.  So the setter doc can remain and then the `qmlproperty` doc added to the bottom of the file.